Specs/Scene/ModelSpec.js

```diff
@@ -85,7 +87,10 @@ defineSuite([
     var texturedBoxCustomUrl = './Data/Models/Box-Textured-Custom/CesiumTexturedBoxTest.gltf';
     var texturedBoxKhrBinaryUrl = './Data/Models/Box-Textured-Binary/CesiumTexturedBoxTest.glb';
     var boxRtcUrl = './Data/Models/Box-RTC/Box.gltf';
-    var boxesEcefUrl = './Data/Models/Boxes-ECEF/ecef.glb';
+    var boxEcefXUpUrl = './Data/Models/Box-ECEF/ecef-x-up.gltf';
```

Comment:
Instead of having three copies of the data, is it reasonable to have one copy (with squashed commits) and just have the test do a small change to the model at load?  This will be less test data and easier to update to glTF 2.0.